pom.xml

```diff
@@ -1,67 +1,67 @@
-        </configuration>
-      </plugin>
-
-    </plugins>
-    <pluginManagement>
-      <plugins>
-        <plugin>
-          <groupId>org.springframework.boot</groupId>
-          <artifactId>spring-boot-maven-plugin</artifactId>
-          <version>${spring.boot.version}</version>
-        </plugin>
-      </plugins>
-    </pluginManagement>
-  </build>
-</project>
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.devonfw</groupId>
+    <artifactId>maven-parent</artifactId>
+    <version>6</version>
+  </parent>
+  <groupId>com.devonfw.java.dev</groupId>
+  <artifactId>devon4j</artifactId>
+  <version>dev-SNAPSHOT</version> <!-- This POM will never be released -->
+  <packaging>pom</packaging>
+  <name>${project.artifactId}</name>
+  <description>Java stack of devonfw.com (devon4j): develop Java apps standardized, faster and with less risk.</description>
+  <inceptionYear>2014</inceptionYear>
+
+  <modules>
+    <module>boms</module>
+    <module>modules</module>
+    <module>starters</module>
+    <module>templates</module>
+    <module>documentation</module>
+  </modules>
+
+  <properties>
+    <revision>dev-SNAPSHOT</revision>
+    <github.repository>devon4j</github.repository>
+    <devon4j.version>${revision}</devon4j.version>
+    <spring.boot.version>2.4.4</spring.boot.version>
+    <!-- Spring boot and spring cloud version has to match -->
+    <spring.cloud.dependencies.version>2020.0.0</spring.cloud.dependencies.version>
+    <jackson.version>2.12.2</jackson.version> <!-- Overriding Jackson for fixing vulnerabilities -->
+    <guava.version>30.1.1-jre</guava.version>
+    <junit.version>5.7.1</junit.version>
+    <cxf.version>3.4.3</cxf.version>
+    <mmm.util.version>8.7.0</mmm.util.version>
+    <slf4j.version>1.7.30</slf4j.version>
+    <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+    <sonar.organization>devonfw</sonar.organization>
+  </properties>
+
+  <build>
+    <plugins>
+
+      <!-- avoid version in local war files, exclude development properties from WARs -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-war-plugin</artifactId>
+        <configuration>
+          <packagingExcludes>WEB-INF/classes/config/application.properties,*.jsp</packagingExcludes>
+          <warName>${project.artifactId}</warName>
+        </configuration>
+      </plugin>
+
+    </plugins>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.springframework.boot</groupId>
+          <artifactId>spring-boot-maven-plugin</artifactId>
+          <version>${spring.boot.version}</version>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+  </build>
+</project>

```

Comment:
I have restored the file but still it is showing in files changed.

Reply:
Current `pom.xml` has linux EOL style:

```
cat -ben pom.xml
     1	<?xml version="1.0" encoding="UTF-8"?>$
     2	<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"$
     3	  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">$
     4	  <modelVersion>4.0.0</modelVersion>$
....
```

Yours has windows encoding. Maybe something with your git setup or the editor you are using to edit XML files.
```
cat -ben pom.xml.ayushi
     1	<?xml version="1.0" encoding="UTF-8"?>^M$
     2	<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"^M$
     3	  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">^M$
     4	  <modelVersion>4.0.0</modelVersion>^M$
...
```

We once had such changes that added a Byte-Order-Mark to a `pom.xml` that was not easily visible in the diff but broke the release as maven pull-parser can not handle BOM. That is why am a little picky about such things.